tests/migrations/test_loader.py

```diff
@@ -205,6 +205,22 @@ def test_marked_as_unmigrated(self):
         self.assertEqual(migration_loader.migrated_apps, set())
         self.assertEqual(migration_loader.unmigrated_apps, {'migrated_app'})
 
+    @override_settings(
+        INSTALLED_APPS=['migrations.migrations_test_apps.migrated_app'],
+        MIGRATION_MODULES={'migrated_app': 'missing-module'},
+    )
+    def test_explicit_missing_module(self):
+        """
+        If a MIGRATION_MODULES override points to a missing module the error
+        raised during the importation attempt should be propagated unless
+        `ignore_no_migrations=True`.
```

Comment:
The `ignore_no_migrations` flag is used by the `makemigrations` command to allow initial migrations to be created.